src/plugins/orangehrmLDAPAuthenticationPlugin/Api/Traits/LDAPDataMapParamRuleCollection.php

```diff
@@ -67,7 +75,7 @@ protected function getParamRuleCollection(): ParamRuleCollection
             $this->getValidationDecorator()->notRequiredParamRule(
                 new ParamRule(
                     self::PARAMETER_EMPLOYEE_ID,
-                    new Rule(Rules::EMAIL),
+                    new Rule(Rules::STRING_TYPE),
```

Comment:
@SandeepaOHRM  This is not email validation. this is email attribute field so i changed it